wagtail/admin/tests/test_account_management.py

```diff
@@ -112,7 +112,7 @@ def test_logout(self):
         self.login()
 
         # Get logout page
-        response = self.client.get(reverse("wagtailadmin_logout"))
+        response = self.client.post(reverse("wagtailadmin_logout"))
```

Comment:
Do we need to change the behaviour of the logout option in the menu to match? Looks like this is currently just a plain link.